ui/app/models/kv/metadata.js

```diff
@@ -15,7 +15,7 @@ const validations = {
     { type: 'length', options: { min: 1, max: 16 }, message: 'You cannot go over 16 characters.' },
   ],
 };
```

Comment:
order matters when rendering. We wanted customMetadata to display at the top of the form, so I moved the ordering around.